op-chain-ops/genesis/layer_one.go

```diff
@@ -104,8 +104,6 @@ func BuildL1DeveloperGenesis(config *DeployConfig) (*core.Genesis, error) {
 		"initialize",
 		big.NewInt(0),
 		uint642Big(uint64(config.L1GenesisBlockTimestamp)),
-		config.L2OutputOracleProposer,
-		config.L2OutputOracleOwner,
```

Comment:
This becomes a bit sticky because now the name of the config value should change, otherwise it is stale and can be confusing

Reply:
If we change the config value name then its going to be a large diff

Reply:
Ok I changed it